lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
Chiming in on Avatar vs Drupal user picture -- assuming nothing's changed over the past year or so -- yep the only place we use the Drupal `user.picture` is when we have a MCC - our staff uploads an image we prepare with the Member on a transparent background. Deprecating the Drupal `user.picture` would be ideal but likely requires tweaks to the Member Created designs... if we even use them anymore? 

`destroy` 🔥 🚒